docs/control-structures.rst

```diff
@@ -598,7 +598,8 @@ function calls which will cause a Panic.
 A Panic exception is generated in the following situations.
 The error code supplied with the error data indicates the kind of panic.
 
-#. 0x01: If you call ``assert`` with an argument that evaluates to false.
+#. 0x00: Used for generic compiler inserted panics.
+#. 0x01: If you call ``assert`` with an argument that evaluates to false. Also used for generic compiler inserted assertions.
```

Comment:
Just out of curiosity - where does it insert generic assertions and where generic panics, and what's the difference?

Reply:
I haven't actually checked if `0x01` is used, copied this description from `ErrorCodes.h`.

Reply:
Code 0x01 should only be used by `assert()`. It is still generic / unspecified, but not compiler-generated per se.